spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -367,14 +369,17 @@ def step_text_in_row(row)
     page.find(".govuk-table__body .govuk-table__row:nth-child(#{row}) .step-list__title").text
   end
 
-  def but_I_cannot_edit_delete_or_reorder_steps
-    expect(page).to_not have_link("Reorder")
+  def but_I_cannot_edit_or_delete_steps
     all('.govuk-table.step-list .govuk-table__cell').each do |step|
       expect(step).not_to have_link("Edit")
       expect(step).not_to have_link("Delete")
     end
   end
 
+  def and_I_cannot_reorder_steps
+    expect(page).to_not have_link("Reorder", exact: true)
```

Comment:
Not sure why previous commit passes tests but current commit needs `exact: true` 🤔